scipy/stats/_fit.py

```diff
@@ -164,10 +192,78 @@ def plot(self, ax=None):
             ax.plot(self._data, np.zeros_like(self._data), "*",
                     label='Data', color='C1')
 
-        ax.set_title(f"{self._dist.name} Fit")
+        ax.set_title(f"`{self._dist.name}` Fit Histogram")
```

Comment:
How about rendering in teletype?

Reply:
I made it render in teletype. I'll wait to push in case there are other comments.

![image](https://user-images.githubusercontent.com/6570539/188805363-47d89b1d-79a5-4489-9fbb-61576bb354c7.png)